### ISSUE TITLE
Cross-platform bundling with electron-builder 21 fails (API breaking change)

### ISSUE THREAD

**Issue by lbssousa** [quasarframework/quasar#4691]
**Describe the bug**
I have a new Electron-based Quasar project in a GitLab private repository, using electron-builder to bundle it in a distributable format. I've configured GitLab CI to build my project for Linux and Windows, for both 32 and 64-bit architectures, using **electronuserland/builder** Docker image (**electronuserland/builder:wine** for Windows builds) from Docker Hub.

I've just noticed that only **Linux 64-bit** builds succeed in my CI. Any **Windows** builds fail with the following error message:
```
(node:48) UnhandledPromiseRejectionWarning: Error: Unknown option "platform"
```

And my **Linux 32-bit** builds fail with the following error message:
```
(node:48) UnhandledPromiseRejectionWarning: Error: Unknown option "arch"
```
May it be due to a possible breaking change in electron-builder API used by Quasar Framework? @develar, could you bring us some light here?

**Platform (please complete the following information):**
@quasar/app version: 1.0.4
electron-builder version: 21.0.5

**Comment by develar** [comment]
Please see https://github.com/electron-userland/electron-builder/releases/tag/v21.0.3

> remove deprecated --arch, --platform, --draft, --prerelease

**Comment by lbssousa** [comment]
Got it! I'm sending a PR fixing this API right now. Thanks!

**Comment by nothingismagick** [comment]
Awesome work you two! @lbssousa @develar

**Comment by rstoenescu** [comment]
Fix will be available in quasar/app v1.1.0.
